src/fiat-connect-client.ts

```diff
@@ -39,12 +39,12 @@ const fetch = fetchCookie(nodeFetch)
 
 export class FiatConnectClient implements FiatConnectApiClient {
   config: FiatConnectClientConfig
-  signingFunction: (message: string) => Promise<string>
+  signingFunction?: (message: string) => Promise<string>
```

Comment:
Not sure I agree this should be optional. Seems easy to forget to pass it in if it's optional, and then you might be surprised by runtime errors later on.

An alternative could be to pass in a dummy signing function if you didn't care about actually signing any data. Or creating a subclass like QuoteOnlyFCClient that does this, or something similar, under the hood.